addon/components/license-list/component.js

```diff
@@ -0,0 +1,75 @@
+import Ember from 'ember';
+import layout from './template';
+
+const defaultCategories = {
+    Content: name => name.indexOf('CC') !== -1,
+    'Code - Permissive': name => name.indexOf('MIT') !== -1 || name.indexOf('Apache') !== -1 || name.indexOf('BSD') !== -1,
+    'Code - Copyleft': name => name.indexOf('GNU') !== -1 && name.indexOf('Lesser') === -1
```

Comment:
Use `includes` instead of `indexOf` to see if a string is in another string, ex. `name.includes('CC')`